src/vanilla/store.ts

```diff
@@ -793,20 +793,24 @@ type Store = ReturnType<typeof createStore>
 
 let defaultStore: Store | undefined
 
+if (import.meta.env?.MODE !== 'production') {
+  if (typeof (globalThis as any).__NUMBER_OF_JOTAI_INSTANCES__ === 'number') {
+    ++(globalThis as any).__NUMBER_OF_JOTAI_INSTANCES__
+  }
+  ;(globalThis as any).__NUMBER_OF_JOTAI_INSTANCES__ = 1
```

Comment:
```diff
} else {
  (globalThis as any).__NUMBER_OF_JOTAI_INSTANCES__ = 1
}
```

Reply:
On, no!!! How did we miss it! Thanks.
Would you like to open a PR?